tests/test_api_models.py

```diff
@@ -23,7 +23,17 @@ def test_api_creation(api_data):
         cache_schema=False,
     )
     assert isinstance(ocx, Api)
-    assert True
+
+
+def test_retries_set_on_client_session(api_data):
+    ocx = Api(
+        api_key="1eab4217d30d42849dbde0cd1bb94e39",
+        base_url="http://localhost:3000",
+        cache_schema=False,
+    )
+
+    assert ocx._session.adapters["http://"].max_retries.total == 3
```

Comment:
nit: it'd be nice to also test that it's set for https (that's the one we care the most about) and that `ocx._client.session` has retries set (that object path is used by the upload module & other places in the library)